### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by rzaliznyi**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -16,4 +16,16 @@
...
+
+mutable_values = (list, dict, set)
+immutable_values = (int, str, tuple, type(None), float, bool)
+
+for el in values:
+    if isinstance(el, mutable_values):
+        sorted_variables["mutable"].append(el)
+
+    elif isinstance(el, immutable_values):
+        sorted_variables["immutable"].append(el)
```
Comment (Dimosphen1) [inline app/main.py:31]: In this task, you just need to fill the `sorted_variables` dictionary with the required values, no additional logic is required.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Comment by rzaliznyi** [comment]
Did my new solution was automatically uploaded or i need to make new pull-request?

**Review by Dimosphen1 — APPROVED** [review]
LGTM, nice work!

**Pull request closed without merging**